### PR TITLE
ci: CRON for packager codes

### PR DESCRIPTION
**Pull request by teolemon**
### What
ci: CRON for packager codes

### CODE REVIEW

File: .github/workflows/packager-codes-refresh.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+name: Update assets
```
Comment (teolemon) [inline .github/workflows/packager-codes-refresh.yml:1]:
```suggestion
name: Packager codes refresh
```

File: .github/workflows/packager-codes-refresh.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+on:
+  # Allow to be run manually
+  workflow_dispatch:
+  schedule:
+  - cron: "0 19 * * *"
```
Comment (teolemon) [inline .github/workflows/packager-codes-refresh.yml:7]: less than daily

File: .github/workflows/packager-codes-refresh.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      id: check-changes
+      uses: mskri/check-uncommitted-changes-action@v1.0.1
+      
+    - name: Create Pull Request
+      if: steps.check-changes.outputs.outcome == failure()
+      id: cpr
+      uses: peter-evans/create-pull-request@v5
+      with:
+        base: develop
+        commit-message: Update assets
```
Comment (teolemon) [inline .github/workflows/packager-codes-refresh.yml:32]:
```suggestion
        commit-message: Update packager codes assets
```

File: .github/workflows/packager-codes-refresh.yml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    if: github.repository_owner == 'openfoodfacts'
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v4
+
+    - name: Upgrade assets
+      run: chmod +x scripts/packager_codes/update_all_packager_codes.sh && scripts/packager_codes/update_all_packager_codes.sh
```
Comment (teolemon) [inline .github/workflows/packager-codes-refresh.yml:20]:
```suggestion
      run: make update_all_packager_codes
```

File: .github/workflows/packager-codes-refresh.yml (hunk 2 above)
Comment (teolemon) [inline .github/workflows/packager-codes-refresh.yml:7]:
```suggestion
  - cron: "0 19 1 * *"
```

File: Makefile [hunk 5]
```diff
@@ -392,6 +392,11 @@ create_external_networks:
 	docker network create --driver=bridge --subnet="172.30.0.0/16" ${COMPOSE_PROJECT_NAME}_webnet \
 	|| echo "network already exists"
 
+
+update_all_packager_codes:
+	@echo "🥫 Dowloading packager codes (production only) …"
+	${DOCKER_COMPOSE} run --rm backend perl /opt/product-opener/scripts/packager_codes/refresh_all_packager_codes.pl
```
Comment (teolemon) [inline Makefile:398]:
```suggestion
	${DOCKER_COMPOSE} run --rm backend perl /opt/product-opener/scripts/packager_codes/update_all_packager_codes.sh
```

**Comment by alexgarel** [comment]
closing in favour of #9295 because a cron- name branch make staging mongodb refresh the data !

**Pull request closed without merging**